cli/src/host.rs

```diff
@@ -203,6 +207,21 @@ impl win32::Host for EnvRef {
         }
     }
 
+    fn create_dir(&self, path: &WindowsPath) -> Result<(), u32> {
+        let path = windows_to_host_path(path);
+        std::fs::create_dir(path).map_err(|e| io_error_to_win32(&e))
+    }
+
+    fn remove_file(&self, path: &WindowsPath) -> Result<(), u32> {
+        let path = windows_to_host_path(path);
```

Comment:
Not for this change, but it's probably worth starting to think about how we ought to model filesystem traversal.  For example can a rogue win32 app (or bug in the emulator) generate paths that make a `rm /r` into a `rm -rf /` outside?

Reply:
Thankfully there's no `rm -r` winapi (implemented, at least) that would make this an "easy" thing to happen. It'd have to recursively call DeleteFile, RemoveDirectory to make that happen.